adsmp/app.py

```diff
@@ -160,3 +217,59 @@ def get_msg_status(self, msg):
                 return 'active'
         else:
             return 'unknown'
+        
+    
+
+    def reindex(self, solr_docs, batch_insert, batch_update, solr_urls):
+        """Sends documents to solr and to Metrics DB.
```

Comment:
Updating Solr and Metrics seems tightly coupled to me.  Imagine the Metrics schema changes.  To push 12 million new records to production metrics (or a test metrics) do I also have to push them to Solr?  In the future, if we rely more on non-Solr based persistent stores this issue will come up more often.